src/curated-corpus/components/LoadExtraButton/LoadExtraButton.tsx

```diff
@@ -1,51 +0,0 @@
-import React from 'react';
-import { Box } from '@material-ui/core';
-import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
-import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward';
-import { Button } from '../../../_shared/components';
-
-interface LoadExtraButtonProps {
-  /**
-   * Which icon to show alongside the label of the button.
-   */
-  arrowDirection: 'up' | 'down';
-  /**
-   * What text to show on the button itself.
-   */
-  label: string;
-  /**
-   * A function to execute when the button is clicked.
-   */
-  onClick: () => void;
-}
-
-/**
- * A 'Load More Results' button, placed below a pageful of results, such as
- * published collections.
- *
- * @param props
- * @constructor
- */
-export const LoadExtraButton: React.FC<LoadExtraButtonProps> = (
-  props
-): JSX.Element => {
-  const { arrowDirection, label, onClick } = props;
```

Comment:
On close inspection, this component was not actually used by anything, so I binned it.